README.md

```diff
@@ -347,6 +347,7 @@ Table of Contents
   * [Discord](https://discord.com/) — Chat with public/private rooms. Markdown text, voice, video, and screen sharing capabilities. Free for unlimited users.
   * [Telegram](https://telegram.org/) — Telegram is for everyone who wants fast and reliable messaging and calls. Business users and small teams may like the large groups, usernames, desktop apps and powerful file sharing options.
   * [Duckly](https://duckly.com/) — Talk and collaborate in real-time with your team. Pair programming with any IDE, terminal sharing, voice, video and screen sharing. Free for small teams.
+  * [Dyte](https://dyte.io) - The most developer-friendly live video & audio SDK, featuring collaborative plugins to enhance productivity and engagement.
```

Comment:
pls provide a short bit of details about what's free

Reply:
Hey @ripienaar, added a sentence about the free tier.
Dyte provides 10,000 minutes of live video/audio usage per month to every organization. You can build your own app (web/mobile/desktop) using Dyte, and the first 10,000 minutes of audio/video conferencing will be free of cost, every month.

Let me know if you'd want me to change the content I've added in the `README.md`.